### PR TITLE
GoogLeNet 2nd polished

### PR DESCRIPTION
**Pull request by goldmermaid**

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-610/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-610/

File: chapter_convolutional-modern/googlenet.md [hunk 1]
```diff
@@ -333,20 +334,20 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
 ## 小结
 
 
-* Inception 块相当于一个有4条线路的子网络。它通过不同窗口形状的卷积层和最大池化层来并行抽取信息，并使用 $1×1$ 卷积层减少通道数从而降低模型复杂度。
-* GoogLeNet将多个设计精细的 Inception 块和其他层串联起来。其中 Inception 块的通道数分配之比是在ImageNet 数据集上通过大量的实验得来的。
+* 初始块相当于一个有4条线路的子网络。它通过不同窗口形状的卷积层和最大池化层来并行抽取信息，并使用 $1×1$ 卷积层减少通道数从而降低模型复杂度。
```
Comment (astonzhang) [inline chapter_convolutional-modern/googlenet.md:337]: 不要翻译Inception块

**Comment by mli** [comment]
Job d2l-zh/PR-610/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-610/

**Comment by mli** [comment]
Job d2l-zh/PR-610/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-610/

**Comment by mli** [comment]
Job d2l-zh/PR-610/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-610/

**Pull request merged**